examples/03_connectivity/plot_rest_clustering.py

```diff
@@ -1,16 +1,37 @@
 """
-Ward clustering to learn a brain parcellation from rest fMRI
+Clustering methods to learn a brain parcellation from rest fMRI
 ====================================================================
 
-We use spatially-constrained Ward-clustering to create a set of
-parcels. These parcels are particularly interesting for creating a
-'compressed' representation of the data, replacing the data in the fMRI
-images by mean on the parcellation.
+We use spatially-constrained Ward-clustering and KMeans to create a set
+of parcels.
+
+In a high dimensional regime, these methods are particularly interesting
+for creating a 'compressed' representation of the data, replacing the data
+in the fMRI images by mean on the parcellation.
+
+On the other way, these methods will also be interesting for learning
```

Comment:
Also, these methods can be interesting to learn... for instance to use in a classification task be controls and disease states